package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@digitalbazaar/vc-commit-tracker",
+  "version": "0.0.1-0",
+  "description": "Retrieves the most recent commit dates from a list of VC specs and test suites.",
+  "type": "module",
+  "main": "lib/index.js",
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com/"
+  },
```

Comment:
@JSAssassin you should add yourself to the `contributors` array: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#people-fields-author-contributors